templates/snap-details.html

```diff
@@ -29,14 +29,28 @@
           <div class="p-media-object__details">
             <h1>{{ snap_title }}</h1>
             <p class="p-media-object__content--large">{{ publisher }}</p>
-
-            <p class="p-media-object__content--large">
-              {% if is_linux %}
-                <a class="p-button--positive" href="snap://{{ package_name }}" onclick="if(typeof ga !== 'undefined'){ga('gtm1.send', {hitType: 'event', eventCategory: 'Snap details', eventAction: 'Clicked Install button', eventLabel: 'Install {{ package_name }}'})}">Install</a>
-              {% else %}
-                You can install this software on a snap-powered system like <a class="p-link--external" href="https://www.ubuntu.com/download">Ubuntu</a>.
-              {% endif %}
-            </p>
+            <div class="row p-media-object__content--large">
+              <div class="col-6">
+                <div class="p-code-snippet">
+                  <input
+                    class="p-code-snippet__input"
+                    id="snap-install"
+                    value="sudo snap install {{ package_name }}"
+                    readonly="readonly"
+                  />
+                  <button
+                    class="p-code-snippet__action"
+                    id="copy"
+                    data-clipboard-target="#snap-install"
+                    onclick="if(typeof ga !== 'undefined'){ga('gtm1.send', {hitType: 'event', eventCategory: 'Snap details', eventAction: 'Copy CLI install button', eventLabel: 'CLI install {{ package_name }}'})}">
```

Comment:
Would you mind moving the event to the `p-code-snippet` div? Not everyone is using a button to select text.

Reply:
I originally thought about trying to be clever and check whether the text had been highlighted, but that seemed overkill.

Perhaps 2 events, on on the button (as is) and another on the input `onfocus`?

Reply:
I think having 2 events will end up being quite spammy, especially since clicking the copy button is (afaict) moving the focus around. I just did some more testing and it looks like using an `oncopy` event on the `input` will be the less spammy version we can have and will only be triggered on and actual copy to clipboard action. If you agree, please also change the `eventAction` value to "Copy CLI install instructions".

Reply:
Added my proposal, let me know what you think

Reply:
Sorry, github didn't update... I've now implemented the oncopy as you suggested